Source/DataSources/exportKml.js

```diff
@@ -20,14 +20,31 @@ import HeightReference from "../Scene/HeightReference.js";
 import HorizontalOrigin from "../Scene/HorizontalOrigin.js";
 import VerticalOrigin from "../Scene/VerticalOrigin.js";
 import when from "../ThirdPartyNpm/when.js";
-import zip from "../ThirdParty/zip.js";
 import BillboardGraphics from "./BillboardGraphics.js";
 import CompositePositionProperty from "./CompositePositionProperty.js";
 import ModelGraphics from "./ModelGraphics.js";
 import RectangleGraphics from "./RectangleGraphics.js";
 import SampledPositionProperty from "./SampledPositionProperty.js";
 import SampledProperty from "./SampledProperty.js";
 import ScaledPositionProperty from "./ScaledPositionProperty.js";
+import zip from "../ThirdPartyNpm/zip.js";
+import pako from "../ThirdPartyNpm/pako.js";
+
+var DeflateInflate = zip.initShimAsyncCodec(
```

Comment:
This is a placeholder call to an init function that does not use workers but I imagine it's better to use the `pako` ones we already have (see my `pako` comment). 

Should the call to the zip configuration function live in the ThirdParty/npm file?